src/jsutils/__tests__/inspect-test.ts

```diff
@@ -169,7 +169,7 @@ describe('inspect', () => {
     (Foo.prototype as any)[Symbol.toStringTag] = 'Bar';
     expect(inspect([[new Foo()]])).to.equal('[[[Bar]]]');
 
-    const objectWithoutClassName = new (function () {
+    const objectWithoutClassName = new (function (this) {
```

Comment:
it's incorrect fix, please return eslint disable comment you removed but change them to disable new rule